### DISCUSSION TITLE
Marine isotope stages (MIS)

### DISCUSSION THREAD

**Discussion by rybesh** [periodo/periodo-data#91] [New period authorities]
These should have some representation in PeriodO.

https://en.wikipedia.org/wiki/Marine_isotope_stage

**Comment by atomrab** [comment]
Agreed. We should work on the chronostratigraphic periods, too, now probably for a newer version than 2017. Do we need to pick up the thread with Cox et al. for this?

https://data.csiro.au/collections/#collection/CIcsiro:41550 (CC-By)
